README.md

```diff
@@ -634,31 +634,27 @@ When running a binary (`pipx run BINARY`), pipx will
 These are all things you can do yourself, but pipx automates them for you. If you are curious as to what pipx is doing behind the scenes, you can always pass the `--verbose` flag to see every single command and argument being run.
 
 ## Contributing
-To develop `pipx` first clone the repository, then create and activate a virtual environment.
+To develop `pipx` first clone the repository, have tox with tox-venv installed somewhere, then run tox.
```

Comment:
Can you add the command to run here, i.e.
```
python3 -m pip install --user tox tox-venv
```

Reply:
Is that "the command" to run here? I imagine there are a number of ways to "install tox", including installing system-wide or using pipx to install it. In fact, "system-wide" is how I do it today and "pipx installed" is how I plan to make it available in my development environments in the future. The PyPI says "pip install tox" is the command. I'd say it's outside of the scope of a given project to prescribe how one should install tox (or Python or pip for that matter), but that the Python ecosystem or the project itself should describe how to install it.

Moreover, `python3` doesn't exist on Windows... so literally every part of that recommendation except for "install tox tox-venv" is an implementation detail. My strong preference would be to leave it to the user (or linked resource) to guide the user on _how_ to achieve install of tox.